File: day03.cpp
#include <iostream>
using namespace std;

void printingAlternateElements(int arr[], int n)
{
    for (int i = 0; i < n;)
    {
        cout << arr[i] << " ";
        i += 2;
    }
}

int main()
{
    int n = 5;
    int arr[] = {1, 2, 3, 4, 5};
    printingAlternateElements(arr, n);
    return 0;
}

File: day03.java

public class day03 {

    public static void main(String[] args) {
        // int n = 5; It can be used when we are inputing the elements from the user
        int[] arr = { 1, 2, 3, 4, 5, 6, 7, 8, 89 };

        printAlternateElements(arr);

    }

    public static void printAlternateElements(int[] arr) {
        int track = 0;
        for (int i : arr) {
            if (track % 2 == 0) {
                System.out.print(i + " ");
            }

            track++;
        }
    }
}